modules/apps/fragment/fragment-impl/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "fragment-impl",
+	"scripts": {
+		"build": "liferay-npm-scripts build",
+		"checkFormat": "liferay-npm-scripts check",
+		"format": "liferay-npm-scripts fix",
```

Comment:
These 2 we don't need, but we might as well keep them :)

Reply:
Replied https://github.com/liferay-frontend/liferay-portal/pull/268#pullrequestreview-472312434